Moonbell crashing server during postinit

### Expected Behavior
To Start up with the server

### Actual Behavior
Crashes the server; had an issue with the "IllegalAccessError" that has since been fixed with the newest update, now giving error saying "ArrayIndexOutofBounds"

### Steps to Reproduce


### Version of Minecraft, Bewitchment, Forge, Patchouli, LLibrary and Baubles used
Latest

### Screenshots encouraged
N/A


### Logs, in the event of a crash
[crash-2019-03-12_22.59.02-server.txt](https://github.com/Um-Mitternacht/Bewitchment/files/2959621/crash-2019-03-12_22.59.02-server.txt)


### If this is a question, developer commentary, or a suggestion, ignore this template

Actual Error:
```
Caused by: java.lang.ArrayIndexOutOfBoundsException
	at java.util.ArrayList.add(ArrayList.java:463)
	at com.bewitchment.common.block.natural.plants.BlockMoonbell.addValidMoonbellBiome(BlockMoonbell.java:45)
```
I believe @CammiePone wrote the Moonbell? Take a look at this please.

--- a/src/main/java/com/bewitchment/common/Bewitchment.java
+++ b/src/main/java/com/bewitchment/common/Bewitchment.java
@@ -203,7 +203,7 @@ public class Bewitchment {
 	@SuppressWarnings("deprecation")
 	@EventHandler
 	public void postInit(FMLPostInitializationEvent evt) {
-		BiomeDictionary.getBiomes(BiomeDictionary.Type.FOREST).parallelStream().filter(b -> BiomeDictionary.hasType(b, BiomeDictionary.Type.DENSE)).forEach(b -> {
+		BiomeDictionary.getBiomes(BiomeDictionary.Type.FOREST).stream().filter(b -> BiomeDictionary.hasType(b, BiomeDictionary.Type.DENSE)).forEach(b -> {
 			BlockMoonbell.addValidMoonbellBiome(b);
 		});
 		if (entityConfig.hasChanged()) {

